playground/html/__tests__/html.spec.ts

```diff
@@ -242,3 +250,9 @@ describe.runIf(isServe)('invalid', () => {
     expect(content).toBeTruthy()
   })
 })
+
+test.runIf(isServe)('importmap', () => {
```

Comment:
```suggestion
test('importmap', () => {
```
I think we should run this test for build, too. (I think this is the reason why we missed the build not working)